dashboard/cron.py

```diff
@@ -656,6 +648,9 @@ def do(self):
         if courses_added_during_cron:
             logger.warning(f'During the run, users added {len(courses_added_during_cron)} course(s): {courses_added_during_cron}')
             logger.warning(f'No data was pulled for these courses.')
+            # Set all of the courses to have been updated now (this is the same set update_course runs on)
```

Comment:
I don't think this should be indented?

Reply:
Yeah, fixed, I think I was thinking of putting it in here for some reason but it didn't make sense in the end, just didn't fix the comment.

Reply:
Doesn't look fixed?